common/src/main/java/xyz/wagyourtail/jsmacros/client/api/classes/TextBuilder.java

```diff
@@ -101,6 +104,17 @@ public TextBuilder withFormatting(boolean underline, boolean bold, boolean itali
         self.styled(style -> style.withFormatting(formattings.toArray(new Formatting[0])));
         return this;
     }
+
+    /**
+     * @param formattings the formattings to apply
+     * @return self for chaining.
+     *
+     * @since 1.8.4
+     */
+    public TextBuilder withFormatting(FormattingHelper... formattings) {
+        self.styled(style -> style.withFormatting(Arrays.stream(formattings).map(FormattingHelper::getRaw).toArray(Formatting[]::new)));
+        return this;
+    }
```

Comment:
I'm not sure what you mean. How would 16 map to both gold and bold? I think it would be better to split the `withFormatting` function into 5 parts (without deleting the old method) for each option.

Reply:
thats a lower case L hot a 1